### DISCUSSION TITLE
请问这个软件开发使用的UI库是什么？

### DISCUSSION THREAD

**Discussion by SekiBetu** [HMBSbige/BilibiliLiveRecordDownLoader#42] [Q&A]
挺漂亮的

**Comment by SekiBetu** [comment]
没事了，源码里看到了一个ModernWpf，搜了下应该是这个项目吧？ https://github.com/Kinnara/ModernWpf

**Reply by HMBSbige** [reply]
确实，准确的说是 Fluent Design

以后 Windows 的软件都这种设计
